app/src/components/ProjectFilters.vue

```diff
@@ -35,6 +35,15 @@
           />
         </div>
 
+        <div class="section">
+          <p class="font-display font-medium text-sm mb-2 px-2">Expertise Level</p>
+          <CheckboxGroup
+            prefix="expertise_level"
+            :keys="['Beginner', 'Intermediate', 'Advanced']"
+            :values="['beginner', 'intermediate', 'advanced']"
```

Comment:
I did all caps for these, we'll probably want to do that here too

Reply:
Acknowledged.